x/evm/ante/fee.go

```diff
@@ -94,7 +94,11 @@ func (fc EVMFeeCheckDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate b
 
 	// calculate the priority by dividing the total fee with the native gas limit (i.e. the effective native gas price)
 	priority := fc.CalculatePriority(ctx, txData)
-	ctx = ctx.WithPriority(priority.Int64())
+
+	// only set priority if it is valid
+	if priority.IsInt64() {
```

Comment:
what if priority is not int64? should we set some default priority?

Reply:
It will be default to 0 if not set

Reply:
But maybe we can explicitly set it to 0 again